routing/routing_tests/turns_tts_text_tests.cpp

```diff
@@ -156,6 +156,95 @@ UNIT_TEST(GetTtsTextTest)
   TEST_EQUAL(getTtsText.GetTurnNotification(notification4), "Затем. Поворот налево.", ());
 }
 
+UNIT_TEST(GetTtsStreetTextTest)
+{
+  string const engShortJson =
+      "\
+      {\
+      \"in_300_meters\":\"In 300 meters.\",\
+      \"in_500_meters\":\"In 500 meters.\",\
+      \"then\":\"Then.\",\
+      \"make_a_right_turn\":\"Make a right turn.\",\
+      \"make_a_left_turn\":\"Make a left turn.\",\
+      \"dist_direction_onto_street\":\"%1$s %2$s onto %3$s\",\
+      \"you_have_reached_the_destination\":\"You have reached the destination.\"\
+      }";
+
+  string const jaShortJson =
+      "\
+      {\
+      \"in_300_meters\":\"三百メートル先\",\
+      \"in_500_meters\":\"五百メートル先\",\
+      \"then\":\"その先\",\
+      \"make_a_right_turn\":\"右折です。\",\
+      \"make_a_left_turn\":\"左折です。\",\
+      \"make_a_right_turn_street\":\"右折し\",\
+      \"make_a_left_turn_street\":\"左折し\",\
+      \"dist_direction_onto_street\":\"%1$s%2$s %3$s に入ります\",\
+      \"you_have_reached_the_destination\":\"到着。\"\
+      }";
+
+  string const faShortJson =
+      "\
+      {\
+      \"in_300_meters\":\"ﺩﺭ ﺲﯿﺻﺩ ﻢﺗﺮﯾ\",\
+      \"in_500_meters\":\"ﺩﺭ ﭖﺎﻨﺻﺩ ﻢﺗﺮﯾ\",\
+      \"then\":\"ﺲﭙﺳ\",\
+      \"make_a_right_turn\":\"ﺐﻫ ﺭﺎﺴﺗ ﺐﭙﯿﭽﯾﺩ.\",\
+      \"make_a_left_turn\":\"ﺐﻫ ﭻﭘ ﺐﭙﯿﭽﯾﺩ.\",\
+      \"dist_direction_onto_street\":\"%1$s %2$s ﺐﻫ %3$s\",\
+      \"you_have_reached_the_destination\":\"ﺶﻣﺍ ﺮﺴﯾﺪﻫ ﺎﯾﺩ.\"\
+      }";
+
+  string const arShortJson =
+      "\
+      {\
+      \"in_300_meters\":\"ﺐﻋﺩ ﺙﻼﺜﻤﺋﺓ ﻢﺗﺭ\",\
+      \"in_500_meters\":\"ﺐﻋﺩ ﺦﻤﺴﻤﺋﺓ ﻢﺗﺭ\",\
+      \"then\":\"ﺚﻣ\",\
+      \"make_a_right_turn\":\"ﺎﻨﻌﻄﻓ ﻲﻤﻴﻧﺍ.\",\
+      \"make_a_left_turn\":\"ﺎﻨﻌﻄﻓ ﻲﺳﺍﺭﺍ.\",\
+      \"dist_direction_onto_street\":\"%1$s %2$s ﺈﻟﻯ \u202C %3$s\",\
+      \"you_have_reached_the_destination\":\"ﻞﻗﺩ ﻮﺼﻠﺗ.\"\
+      }";
```

Comment:
nit: I'd prefer to use C++ R"()" macro for string literals without ugly escaping:
https://en.cppreference.com/w/cpp/language/string_literal

Reply:
I don't object but I'm also not going to make that change at this time because I'm only continuing the existing style used by previous contributors in these files:

- platform/platform_tests/get_text_by_id_tests.cpp
- platform/platform_tests/jansson_test.cpp
- routing/routing_tests/turns_sound_test.cpp
- routing/routing_tests/turns_tts_text_tests.cpp
- storage/storage_tests/country_name_getter_tests.cpp

If we want to change how we write json embedded in c, maybe that can be its own cleanup task outside of this PR which I'd really like to push through to completion.

Reply:
Please write new code following the best practices, without looking at some old code/approaches. Otherwise, we would now write code using C++ 2003 standard )

When time comes, someone will refactor it. Or older code will be removed. Or changed if modification is needed.

Reply:
In this case, reading the tests is important. Don't make it hard for reviewers.

Reply:
Is this a blocker, do you want me to refactor the tests before merge?